### PR TITLE
[Derek Song]resume

### PR DESCRIPTION
**Pull request by dereksong88**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,47 @@
-# Jarivs Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
+
+# DEREK_SONG . Jarvis Consulting
```
Comment (jarviscanada) [inline README.md:2]: Derek Song

File: README.md [hunk 2]
```diff
@@ -1,4 +1,47 @@
...
+
+My name is Derek Song, and I am currently working with Jarvis as a data engineer. My role is to utilize Java and Scala programming languages to build applications. I am proficient in Java, Linux, Bash, SQL, RDBMS, JDBC and DAO design pattern. Before that, I have achieved a Master's Degree and Bachelor's Degree in Engineering, and I have strong programming fundamentals and learning skills. Besides, I worked in TS Tech Canada Inc. for three years, where I built very good communication skills and achieved hands-on experience of working with team and contractors. I really enjoyed my work with Jarvis, and I would love to dig much deeper into this area.
+
+## Skills
+**Proficient:** Java, Maven, Bash, SQL, Agile/Scrum, Docker, RDBMS, JDBC, DAO, Git/GitHub, Microservice, Postman
+
+**Competent:** Spring and SpringBoot framework, Oracle
```
Comment (jarviscanada) [inline README.md:9]: Please run a spelling and grammar check using https://www.grammarly.com/ if necessary

File: resume/.resume.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
+
+# John Smith . Jarvis Consulting
+
+Donec fermentum, lectus tempor lobortis egestas, dolor dolor rhoncus justo, id pharetra odio ligula quis odio. Morbi blandit id quam eget porttitor. Integer at ullamcorper dolor, at egestas turpis. Fusce ultricies justo tellus, a congue augue aliquet sit amet. Quisque ultrices dolor eu turpis malesuada interdum. Suspendisse in nisl et massa laoreet sodales. Nulla quis lectus sit amet velit finibus feugiat eget nec quam. Praesent eget nibh et diam mollis venenatis. Mauris mi eros, varius in elit ut, ornare commodo magna. Nullam lacinia sapien pulvinar cursus tincidunt. Nam neque magna, accumsan a mi vel, tempor tristique nisl. Maecenas accumsan dignissim.
```
Comment (jarviscanada) [inline resume/.resume.md:4]: git remove resume/.resume.md

File: springboot/.idea/.gitignore [hunk 4]
```diff
@@ -0,0 +1,2 @@
+# Default ignored files
```
Comment (jarviscanada) [inline springboot/.idea/.gitignore:1]: git rm springboot/.idea/.gitignore

File: README.md [hunk 5]
```diff
@@ -1,4 +1,47 @@
...
+## Skills
+**Proficient:** Java, Maven, Bash, SQL, Agile/Scrum, Docker, RDBMS, JDBC, DAO, Git/GitHub, Microservice, Postman
+
+**Competent:** Spring and SpringBoot framework, Oracle 
+
+**Familiar:**: Spark, Scala, Hive, Kafka, Nifi
+
+## Development Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_derek](https://github.com/jarviscanada/jarvis_data_eng_derek)
```
Comment (jarviscanada) [inline README.md:15]: please read the instruction carefully. you need to create the `feature/resume` branch from the master. Please recreate a feature/resume2 branch

git checkout master
git checkout -b feature/resume2
#update your resume me and pdf
#commit 
#master PR

**Review by jarviscanada — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (dereksong88) [reply README.md:2]: @jarviscanada Hello Edward, I have created a new branch feature/resume2 upon request, and corrected all the mistakes. A new pull request is posted in my 360 page. Thank you so much.

**Pull request closed without merging**